doc/filters/data_uri.rst

```diff
@@ -16,7 +16,7 @@ The ``data_uri`` filter generates a URL using the data scheme as defined in
     {# also works with plain text #}
     {{ '<b>foobar</b>'|data_uri(mime="text/html") }}
 
-    {# add some extra parameters #}
+    {# we can add any extra parameters we could put in an url #}
```

Comment:
I feel the "we" is more used for internal / algorithm explanations, and "you" to illustrate user need / usages... wdyt ?  

(not as binary in fact, but i feel this is the general idea)

Reply:
You're right, I changed it